src/openrct2/ride/CableLift.cpp

```diff
@@ -325,7 +323,8 @@ bool Vehicle::CableLiftUpdateTrackMotionBackwards()
             SetTrackDirection(output.begin_direction);
             SetTrackType(output.begin_element->AsTrack()->GetTrackType());
 
-            if (output.begin_element->AsTrack()->GetTrackType() == TrackElemType::EndStation)
+            if (output.begin_element->AsTrack()->GetTrackType() == TrackElemType::EndStation
+                || output.begin_element->AsTrack()->GetTrackType() == TrackElemType::BlockBrakes)
```

Comment:
Needs to account for diagonal block brakes.

Reply:
I didn't check for a diagonal brake because there's no diagonal cable lift hill that could go with it.

It is possible to hack a cable lift hill with turns, so it wouldn't hurt to check, but I think the vehicle update code would also need to change to handle the diagonal case properly.

On the other hand, if this is merged without that check, and then a diagonal cable lift piece is added later, it technically could break stuff, so I think I will need to do some testing to make sure it handles the diagonal case properly.